user_manual/changes/T510-001.yaml

```diff
@@ -0,0 +1,7 @@
+type: bugfix
```

Comment:
Please add formatting to find_all_references (double back quotes)

Reply:
Thanks, fixed!